packages/utils/transactionFactory/weiroll/index.ts

```diff
@@ -0,0 +1,17 @@
+// Copied from weiroll repo: https://github.com/EnsoFinance/enso-weiroll.js/blob/25081c263663c035cc773b7d90dfc352b7d7a541/src/planner.ts
```

Comment:
Can you double check if we really copied from here? Think that we changed later.